Branding & Audience section (#16)
osoc24/doco: project/audience.md

---
title: For whom?
next:
    link: /loama/index
    text: 'Loama'
---
# For whom?

> The LOAMA project is created for users with a technological knowledge, but we've focused on several factors while designing the application:

Reviewer: ```suggestion
The LOAMA project is created for users with a technological background, but we've focused on several factors while designing the application:
```

- **Ease of use**: LOAMA was designed with ease of use in mind by reducing & simplifying the steps that need to be taken to assign permissions. 

> - **Accessibiliy**: In order to make LOAMA as accessible as possible, we omitted using technical terms to describe function. Instead, by using a combination of icons, simple sentences and colors, we indicate the status of files and divert attention of the user.

Reviewer: ```suggestion
- **Accessibiliy**: To make LOAMA as accessible as possible, we omitted using technical terms to describe functions. Instead, by using a combination of icons, simple sentences and colours, we indicate the status of files and divert the attention of the user.
```

By using the information gained during the [brand creation](/project/brand.md), we made a persona to guide us during the design process of [Loama](/loama/index.md):

> ## Jermery

Reviewer 2: Jeremy?

Author: Jeremy has been deleted :cry:

### Basic info
- **Age**: 24 years
- **City**: Brussels
- **Occupation**: Tech journalist at Data News
- **Gender**: Male
- **Education**: Bachelor in Journalism

### Pains & Problems
> What are the problems the user encounters?
#### LOAMA
- I can't see what permissions are configured for the files in my pod
- I don't see who can access my files and which permissions they have
- What do the permissions actually mean within my Solid Pod?

> - Why should I use Solid over other technologies or just having a company have my data?

Reviewer: ```suggestion
- Why should I use Solid over other technologies or just letting a company have my data?
```

#### Solid-enabled apps

> - What do the permissions actually mean within my Solid Pod?

Reviewer 2: repeated from line 29

- How do I specify which data an application may use?

### Needs
> What does the users want to do within the app?
#### LOAMA

> - I want to see what data an person, company or application can access

Reviewer 2: ```suggestion
- I want to see what data a person, company or application can access
```

> - I want to see what data an person, company or application can access

Reviewer: ```suggestion
- I want to see what data a person, company or application can access
```

- I want to see when my data was accessed
- I want to revoke or allow acess to my data

> - I want to understand what the permissions means and the effect of when I allow them

Reviewer 2: ```suggestion
- I want to understand what the permissions mean and the effect of allowing them
```

> - I want to understand what the permissions means and the effect of when I allow them

Reviewer: ```suggestion
- I want to understand what the permissions mean and the effect of when I allow them
```

#### Solid-enabled apps
- I want to use my data in my Solid pod across different platforms
- I want to sign on using my Solid pod and allow the application to use my data

### Motivation
> Why does a user wants the things he wants?

- I want to be in control of my own data
- I want to share the same data across platforms

> - I want to easily disabled shared access to sensitive information

Reviewer 2: ```suggestion
- I want to easily revoke access to sensitive information
```

> - I want to easily disabled shared access to sensitive information

Reviewer: ```suggestion
- I want to easily disable shared access to sensitive information
```
